app/views/layouts/_nav.html.haml

```diff
@@ -3,7 +3,11 @@
     %h3 Pad Crash
   %section.nav_right
     %ul
-      %li
-        = link_to("Create Account", '#')
-      %li
-        = link_to("Log In", '#')
+      - if session[:current_user]
```

Comment:
@njgheorghita What would you think about making a `current_user` method in the controller and providing a helper method instead of reaching for the session?